client/views/movie.html

```diff
@@ -9,7 +9,7 @@
     </div>
     <div class="uk-width-xxlarge content">
       <div class="uk-width-3-4">
-              <b>Directed by</b> {{ movie.directors }}
+              <b>Directed by</b> {{ movie._directors }}
```

Comment:
A media can have one or more director**s**. So movie._directors is an array. It would be better to iterate over it.

Notice that currently api/movie/<movie_id> return a json like:

{
    "isBoxOffice": false,
    "overview": "Chronicles the experiences of a formerly successful...",
    "_directors": [
        "5af77d151a60bd4a049fd5ae",
        ...
    ],
    "_actors": [
        "5af779ea1a60bd4a049fd4eb",
        ...
    ],
    "genres": [
        "Crime",
        ...
    ],
    "__t": "Movie",
    "_id": "5af77efa1a60bd4a049fd61d",
    "poster": "https://ia.media-imdb......jpg",
    "release_date": "1994-01-01T00:00:00.000Z",
    "name": "The Shawshank Redemption",
    "imdb_id": "0111161",
    "__v": 0
}

We may need to create a task to return every actor/director as a json and not an id. But yet we should iterate over movie._director and not just print.